.github/workflows/ci.yml

```diff
@@ -468,46 +485,46 @@ jobs:
 
       - name: Upload ESP32 release firmware artifacts
         if: |
-          github.event_name == 'release' &&
+          (github.event_name == 'release' || env.IS_FIX_RELEASE) &&
           github.repository_owner == 'toitlang'
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: build/firmware-esp32.gz
-          tag: ${{ github.event.release.tag_name }}
+          tag: ${{ env.TOIT_VERSION }}
           overwrite: true
 
       - name: Upload ESP32-C3 release firmware artifacts
         if: |
-          github.event_name == 'release' &&
+          (github.event_name == 'release' || env.IS_FIX_RELEASE) &&
           github.repository_owner == 'toitlang'
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: build/firmware-esp32c3.gz
-          tag: ${{ github.event.release.tag_name }}
+          tag: ${{ env.TOIT_VERSION }}
           overwrite: true
 
       - name: Upload ESP32-S2 release firmware artifacts
         if: |
-          github.event_name == 'release' &&
+          (github.event_name == 'release' || env.IS_FIX_RELEASE) &&
```

Comment:
Could this be combined so we always have something like `env.BUILD_RELEASE` set when we should build the release? If the `github.event_name` is `'release'` it should default to true -- otherwise, it can be overwritten.

Reply:
done.